exercise30_v2.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+LIST_OF_NUMBERS = list(range(0, 11))
+
+def create_multiplication_table(numbers):
+    """Create a table with numbers and product"""
+
+    total_products = []
```

Comment:
@paulghaddad I updated the variable name to `total_products` instead of `formula` b/c it's really a list of all of the products, not a formula. Sound good?